fab/pillows/swiss.yml

```diff
@@ -5,3 +5,5 @@ exclude_pillows:
   - ReportCaseToElasticsearchPillow
   - ReportXFormToElasticsearchPillow
   - CaseSearchToElasticsearchPillow
+  - kafka-ucr-static
+  - kafka-ucr-main
```

Comment:
sounds good. also sounds fine to remove the UCR queues for now as well though wonder if this will confuse us at some point down the road...